test/README.md

```diff
@@ -1,7 +1,29 @@
+This folder contains testing infrastructure for the pipelines in the skylab repository.
+
+**Description of Tests**
+- optimus: Tests related to the single-cell RNA-seq version of Optimus
+  - 4kpbmc: Runs the full 4kpbmc dataset with the sc variant of Optimus. Checksums are not provided (manual)
+  - pr: Run a small dataset derived from the 4k PBMC dataset with Optimus. It performs delta testing on the output matrix
```

Comment:
Question- what does pr mean in this context?

Reply:
Yes, also included in intro